torchao/quantization/subclass.py

```diff
@@ -592,3 +604,263 @@ def to_qtensor_components(cls, input_float, groupsize=128, inner_k_tiles=8):
         )
         int_data = aten._convert_weight_to_int4pack(input_int4x8, inner_k_tiles)
         return int_data, scales_and_zeros, False, groupsize, inner_k_tiles
+
+
+class AffineQuantizedTensor(torch.Tensor):
+    """
+    Base affine quantized tensor subclass. When the from_float method is used,
+    to create an instance of any AffineQuantizedTensor
+
+    The shape and dtype of the tensor subclass represent how the tensor subclass looks externally,
```

Comment:
What does externally mean here? Do you mean to say in export?

Reply:
What happens when you call `dtype` and `shape` on an instance of `AffineQuantizedTensor`. I agree the wording is confusing.

Reply:
yeah, this means what it looks like for autograd engine mostly I think